tests/test_hdfio.py

```diff
@@ -59,3 +61,5 @@ def tearDown(self):
 
     test_suite = unittest.TestSuite([unittest.TestLoader().loadTestsFromTestCase(test_hdf)])
     unittest.TextTestRunner(verbosity=2).run(test_suite)
+    os.remove('test.hdf')
```

Comment:
Unfortunately this doesn't help when running with pytest. It looks like it can't go within tearDown because setUp/tearDown is done for every test, so the file is gone after the first test.

Reply:
The updated test set unfortunately doesn't work with the parallel test runner (try with `pytest -n 2 -v -k hdf`)

Reply:
@dwfncar are you satisfied with that resolution?

Reply:
Yes